UnrealPlugin/ThirdPersonMPGSDKSetup.md

```diff
@@ -215,6 +213,7 @@ If you already **had** an Init() function, go to check in [YourGameInstanceClass
 
 ```cpp
 #if UE_SERVER
+    UGSDKUtils::SetDefaultServerHostPort();
```

Comment:
does it work for container based servers as well?

Reply:
Yes, we tested it all the ways.

Reply:
Yes, will update PR description